Gemfile

```diff
@@ -115,7 +115,7 @@ end
 
 group :test do
   gem 'webmock'
-
+  gem 'capybara-screenshot'
```

Comment:
I was trying to squeeze this gem in master for a while now :) Will create new PR!